src/pages/SignPage/KakaoLogin/KakaoLogin.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+const KakaoLogin = () => {
+  const { Kakao } = window;
+  const [login, setLogin] = useState(false);
+
+  const handlelogout = () => {
+    setLogin(!login);
+    //  localStorage.removeItem('kakao_faab735225616551bbbcc9852afc8a06');
+    console.log('로그아웃 버튼을 눌렀어!');
+  };
+
+  const handlelogin = async () => {
+    Kakao.Auth.login({
+      scope: 'profile_nickname , account_email',
+      persistAccessToken: false,
+      success: await function (response) {
+        axios
+          .post('http://10.58.4.22:8000/users/signin', {
+            body: JSON.stringify({ access_token: response.access_token }),
+          })
+          .then(res => {
+            console.log(res);
+          });
+      },
```

Comment:
```suggestion
      success: async function (response) {
        const res = await axios.post('http://10.58.4.22:8000/users/signin', {
            body: JSON.stringify({ access_token: response.access_token }),
          })
        console.log(res);
      },
```
지금 코드 상으로 async 키워드가 의미가 있으려면 위와 같은 방식으로 작성하셔야 합니다. async await 키워드는 Promise 값 앞에서 해당 비동기 작업이 끝나기를 기다렸다가 다음 코드를 실행하는 역할을 합니다. 다음 링크 확인하시고 올바르게 사용해주세요 https://ko.javascript.info/async-await